avoid "Click here ..." and page reload for Contact

dckc opened the pull request.

dckc committed:
avoid "Click here ..."

mythmon commented on index.html:
```diff
@@ -65,11 +65,11 @@
 </ul>
     <div><a href="http://monte.readthedocs.org/en/latest/intro.html">
-        Click here for more information.
+        Introdcution to Monte
```
Typo, introdcution.

dckc replied:
fixed

dckc committed:
same document link

the `index.html#IRC` link causes a page reload from `http://www.monte-language.org/` to `http://www.monte-language.org/index.html`

dckc committed:
typo
